dlt/destinations/impl/athena/factory.py

```diff
@@ -45,6 +47,8 @@ def __init__(
         super().__init__(
```

Comment:
name and environment missing. Also everywhere else

Reply:
I think it was missing only here, i added it.